firefox-ios/Extensions/ActionExtension/ActionViewController.swift

```diff
@@ -0,0 +1,103 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Storage
+import Shared
+
+class ActionViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .clear
+        view.alpha = 0
+
+        getShareItem { [weak self] shareItem in
+            guard let self = self else { return }
+
+            guard let shareItem = shareItem else {
```

Comment:
Can we merge the two guards?
```suggestion
            guard let self = self,
             let shareItem = shareItem else {
```